README.md

```diff
@@ -8,6 +8,33 @@ Demo: https://www.youtube.com/watch?v=HXdQd65CwkU
 ## Status
 WIP but playable.
 
+## Installation
+
+Prebuilt binary releases are available if you are looking to test or play and do not wish to compile yourself.
+| [Releases Page](https://github.com/swoolcock/rush/releases/) |
+| -------------|
+
+**NOTE**: osu!lazer runs custom rulesets only on desktop platforms (Windows, macOS, Linux) for the time being.
+
+The ruleset consists of a single DLL file that you'll need to place in the `rulesets` subdirectory of your osu!lazer installation.
+
+### Installation instructions
+
+1. Navigate to the osu!lazer data directory. You can navigate to it by opening the settings panel on osu!lazer and clicking on the "open osu! folder" button. Alternatively you can directly navigate to the `rulesets` directory via your OS directory explorer at the following locations:
```

Comment:
Replace `You can navigate to it by` with `You can do so by`, and replace `on osu!lazer` with `in osu!lazer`.